cmd/osquery-perf/agent.go

```diff
@@ -26,6 +26,11 @@ import (
 //go:embed *.tmpl
 var templatesFS embed.FS
 
+//go:embed *.software
+var softwareFS embed.FS
+
+var vulnerableSoftware []fleet.Software
```

Comment:
If this is global, then probably better to initialize it on an `init`. Or maybe there's no reason for this to be global, just create it and pass it to `newAgent` :)